charts/k8ssandra/values.yaml

```diff
@@ -537,6 +537,10 @@ monitoring:
     # not have the ServiceMonitor CRD installed on your cluster, set this value
     # to `false`.
     provision_service_monitors: true
+  # -- Deploy the service monitors on a different namespace than the release namespace.
+  # This should only be set if the kube-prometheus stack isn't installed in the release namespace
+  serviceMonitor: {}
```

Comment:
An empty map here suggests that I can specify anything or a ServiceMonitor in its entirety. I would change this based on the properties that you are using - `namespace` and `namespaceSelector`.

Reply:
How can I make it so that the dict exists even if the namespace value isn't provided then?
I've replicated what's right above for `medusa.podStorage` or `medusa.storage_properties`.

Reply:
It should look better now. Let me know what you think.